config/locales/en/errors.yml

```diff
@@ -79,3 +79,28 @@ en:
               blank: "Enter a contacts name"
             contact_email:
               blank: "Enter a contacts email"
+        steps/case_details_form:
+          attributes:
+            ufn:
+              blank: "Enter a unique file number"
+            main_offence:
+              blank: "Enter the name of the offence"
+            main_offence_date:
```

Comment:
can replace with:
```
main_offence_date: *shared_date_errors
```

This is special YAML syntax to avoid replication

Reply:
done